src/rpc/client/ts/src/server.ts

```diff
@@ -71,7 +45,9 @@ export async function startServer(
   rootPath: string,
   version: string,
   port: number = 9000,
-  host: string = '127.0.0.1'
+  host: string = '127.0.0.1',
+  pidfile?: string,
+  nodeCmdPath: string = process.argv[0]
```

Comment:
Swap the order of these 2 arguments.  Convention is to put the optional ones last.